DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_wrong_fcs.py

```diff
@@ -0,0 +1,123 @@
+    """
+    
+    bridge = "br0"
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        print.error("The testbed does not have enough dent with tgen connections")
+        return
+    dent_dev = dent_devices[0]
+    device_host_name = dent_dev.host_name
+    tg_ports = tgen_dev.links_dict[device_host_name][0]
+    ports = tgen_dev.links_dict[device_host_name][1]
+    traffic_duration = 5
+
+    out = await IpLink.add(
+        input_data=[{device_host_name: [
+            {"device": bridge, "type": "bridge"}]}])
+    err_msg = f"Verify that bridge created.\n{out}"
+    assert out[0][device_host_name]["rc"] == 0, err_msg
+
+    out = await IpLink.set(
+        input_data=[{device_host_name: [
+            {"device": bridge, "operstate": "up"}]}])
+    err_msg = f"Verify that bridge set to 'UP' state.\n{out}"
+    assert out[0][device_host_name]["rc"] == 0, err_msg
+    
+    out = await IpLink.set(
+        input_data=[{device_host_name: [
+            {"device": port, "master": "br0", "operstate": "up"} for port in ports]}])
+    err_msg = f"Verify that bridge entities set to 'UP' state and links enslaved to bridge.\n{out}"
+    assert out[0][device_host_name]["rc"] == 0, err_msg
+
+    out = await BridgeLink.set(
+        input_data=[{device_host_name: [
+            {"device": port, "learning": True, "flood": False} for port in ports]}])
+    err_msg = f"Verify that entities set to learning 'ON' and flooding 'OFF' state.\n{out}"
+    assert out[0][device_host_name]["rc"] == 0, err_msg
+
+    address_map = (
+        #swp port, tg port,     tg ip,     gw,        plen
+        (ports[0], tg_ports[0], "1.1.1.2", "1.1.1.1", 24),
+        (ports[1], tg_ports[1], "2.2.2.2", "2.2.2.1", 24),
+        (ports[2], tg_ports[2], "3.3.3.2", "3.3.3.1", 24),
+        (ports[3], tg_ports[3], "4.4.4.2", "4.4.4.1", 24),
+    )
+
+    dev_groups = tgen_utils_dev_groups_from_config(
+        {"ixp": port, "ip": ip, "gw": gw, "plen": plen}
+        for _, port, ip, gw, plen in address_map
+    )
+
+    await tgen_utils_traffic_generator_connect(tgen_dev, tg_ports, ports, dev_groups)
+    
+    list_macs = ["aa:bb:cc:dd:ee:11", "aa:bb:cc:dd:ee:12",
+                 "aa:bb:cc:dd:ee:13", "aa:bb:cc:dd:ee:14"]
+
+    streams = {
+        f"bridge_{dst + 1}": {
+            "ip_source": dev_groups[tg_ports[src]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[dst]][0]["name"],
+            "srcMac": list_macs[src],
+            "dstMac": list_macs[dst],
+            "type": "raw",
+            "protocol": "802.1Q",
+            "bad_crc": True,
+        } for src, dst in ((3, 0), (2, 1), (1, 2), (0, 3))
+    }
+
+    await tgen_utils_setup_streams(tgen_dev, config_file_name=None, streams=streams)
+
+    await tgen_utils_start_traffic(tgen_dev)
+    await asyncio.sleep(traffic_duration)
+    await tgen_utils_stop_traffic(tgen_dev)
+
+    # check the traffic stats
```

Comment:
Check that traffic is not forwarded as well.
But for the fowarding, you should not disable flooding.